geode-core/src/main/java/org/apache/geode/distributed/internal/membership/gms/api/Message.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.distributed.internal.membership.gms.api;
+
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.geode.annotations.Immutable;
+import org.apache.geode.distributed.internal.membership.InternalDistributedMember;
+import org.apache.geode.internal.serialization.DataSerializableFixedID;
+
+/**
+ * The membership module is capable of sending messages to other members of the
+ * cluster. Those messages must implement this interface and must be serializable
+ * using DataSerializableFixedID from the geode-serialization module.
+ * <p>
+ * There are currently two implemementation hierarchies of Message.
+ * One is AbstractGMSMessage, which is used in the membership-module for membership-
+ * related messages. The other is DistributionMessage, which is the base class for
+ * messages in geode-core and other higher-level modules.
+ */
+
+public interface Message extends DataSerializableFixedID {
+
+  /**
+   * Indicates that a distribution message should be sent to all other distribution managers.
+   */
+  @Immutable
+  InternalDistributedMember ALL_RECIPIENTS = null;
```

Comment:
Shouldn't this be MemberIdentifier, not InternalDistributedMember?

Reply:
good catch